### PR TITLE
Add CategoryChoiceField

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: dash/categories/views.py [hunk 1]
```diff
@@ -12,9 +13,9 @@ def __init__(self, *args, **kwargs):
         del kwargs['org']
 
         super(CategoryImageForm, self).__init__(*args, **kwargs)
-        self.fields['category'].queryset = Category.objects.filter(org=self.org)
+        self.fields['category'].queryset = Category.objects.filter(org=self.org).order_by('org__name', 'name')
```
Comment (rowanseymour) [inline dash/categories/views.py]: Ordering by `org__name` won't do anything as you're filtering by a single org

File: dash/categories/views.py [hunk 2]
```diff
@@ -12,9 +13,9 @@ def __init__(self, *args, **kwargs):
...
 
-    category = forms.ModelChoiceField(Category.objects.filter(id__lte=-1))
+    category = CategoryChoiceField(Category.objects.filter(id__lte=-1))
```
Comment (rowanseymour) [inline dash/categories/views.py]: Can use `Category.objects.none()`. Better to not make assumptions about pks being positive.

Also, better to define class fields before `__init__` and other methods.

File: dash/stories/views.py [hunk 3]
```diff
@@ -13,10 +14,12 @@ def __init__(self, *args, **kwargs):
         self.org = kwargs['org']
         del kwargs['org']
         super(StoryForm, self).__init__(*args, **kwargs)
-        qs = Category.objects.filter(org=self.org, is_active=True)
+
+        # We show all categories even inactive one in the dropdown
+        qs = Category.objects.filter(org=self.org).order_by('org__name', 'name')
```
Comment (rowanseymour) [inline dash/stories/views.py]: Same comments as for `CategoryImageForm`

**Comment by rowanseymour** [comment]
Looks fine - just need a little tidying in those two forms

**Comment by norkans7** [comment]
@rowanseymour this is ready for another review

**Comment by rowanseymour** [comment]
Yup

**Pull request merged**